File: PHP/Aksesoris.php
<?php
require_once "Petshop.php";

class Aksesoris extends Petshop {
    protected $jenis_aksesoris;
    protected $bahan_aksesoris;
    protected $warna_aksesoris;

    public function __construct($id = 0, $nama = "", $harga = 0.0, $stok = 0, $foto = "default.jpg", $jenis = "", $bahan = "", $warna = "") {
        parent::__construct($id, $nama, $harga, $stok, $foto);
        $this->jenis_aksesoris = $jenis;
        $this->bahan_aksesoris = $bahan;
        $this->warna_aksesoris = $warna;
    }

    public function getJenis() { return $this->jenis_aksesoris; }
    public function getBahan() { return $this->bahan_aksesoris; }
    public function getWarna() { return $this->warna_aksesoris; }
}
?>

File: PHYTON/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    def __init__(self, id=0, nama="", harga=0.0, stok=0, jenis="", bahan="", warna=""):
        super().__init__(id, nama, harga, stok)
        self._jenis_aksesoris = jenis
        self._bahan_aksesoris = bahan
        self._warna_aksesoris = warna
    
    def set_jenis(self, jenis):
        self._jenis_aksesoris = jenis
    
    def set_bahan(self, bahan):
        self._bahan_aksesoris = bahan
    
    def set_warna(self, warna):
        self._warna_aksesoris = warna
    
    def get_jenis(self):
        return self._jenis_aksesoris
    
    def get_bahan(self):
        return self._bahan_aksesoris
    
    def get_warna(self):
        return self._warna_aksesoris
